### PR TITLE
316 update cellranger modules to latest nf coremodules versions

### PR DESCRIPTION
**Pull request by fmalmeida**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/scrnaseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/scrnaseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -98,7 +98,7 @@ if(params.aligner == "cellranger") {
                 path: "${params.outdir}/${params.aligner}/count",
                 mode: params.publish_dir_mode
             ]
-            ext.args = {"--chemistry ${meta.chemistry} " + (meta.expected_cells ? "--expect-cells ${meta.expected_cells}" : '')}
+            ext.args = {"--chemistry ${meta.chemistry} --create-bam false " + (meta.expected_cells ? "--expect-cells ${meta.expected_cells}" : '')}
```
Comment (grst) [inline conf/modules.config:101]: Why? E.g. RNA velocity analysis relies on this, I'd expect a couple of users complain about this being default all of a sudden.

**Review by grst — COMMENTED** [review]
Any idea why the download test might be failing?

File: conf/modules.config (hunk 1 above)
Comment (fmalmeida) [reply conf/modules.config:101]: New version of cellranger requires this parameter to be used.

Did not know it was used by other workflows. Should I default to true or make a parameter?

**Comment by fmalmeida** [comment]
> Any idea why the download test might be failing?

No idea actually, it just worked fine in gitpod.

**Comment by grst** [comment]
Ok, I just restarted it, maybe it was random 🤞

File: conf/modules.config (hunk 1 above)
Comment (grst) [reply conf/modules.config:101]: I see. A parameter would be nice, but I don't mind setting to true and following up on this separately.

Comment (fmalmeida) [reply conf/modules.config]: Now defaulting to true. Anyways, users can still overwrite if needed as it is a `ext.args`. So, we can consider having a param later upon request.

**Review by apeltzer — APPROVED** [review]

**Pull request merged**